cloudfour/cloudfour.com-patterns
# Patterns/organize icons

## Overview

This PR attempts to address the following tasks identified in #657 : 

> Organization
> 
> Currently our icons sit in one folder, but I think these might benefit from some reorganization:
> 
> - [x] Original icons should sit at the root of the icons directory. These include glyphs like arrows, link, notifications, etc.
> - [x] Icons representing third-party brands should be moved to a "brands" subdirectory. These include icons for publications (A List Apart, Smashing, etc.), social media websites, etc.
> - [x] Our logo should be moved out of icons, possibly one level up.
> 
> This organization should also be reflected in our source files, with separate files for each type of asset.

Once approved, I plan to update the source files to match this updated organization.

## Notes

This PR does not address sizing concerns identified in #657 

> Finally, the logo is used primarily as a branding element (not an icon). Arbitrarily squashing it into our 24×24 grid raises three issues:
> 
> - We have to scale the element up for its primary use case, which is silly.
> - Its coordinates will have more decimal points, which increases the size of the markup required to define the shape.
> - Decimal points may be rounded for performance, which could result in quality loss.

Sizing concerns will be worked on in a separate, forthcoming PR. 

## Questions

1. Do we still use the Flickr icon?
2. Is the canvas tab necessary for icons and brand elements? Can that be disabled if not?

## Screenshots

![Screen Shot 2020-05-07 at 4 37 49 PM](https://user-images.githubusercontent.com/1427548/81354759-4b7ebf00-9081-11ea-8751-249e8638b842.png)

![Screen Shot 2020-05-07 at 4 38 21 PM](https://user-images.githubusercontent.com/1427548/81354762-4cafec00-9081-11ea-9f34-6c5162f9b1b2.png)

## Review & Testing

1. View [deploy preview](https://deploy-preview-691--cloudfour-patterns.netlify.app/)
2. Confirm todos listed above where addressed appropriately
3. Review/discuss this [comment](https://github.com/cloudfour/cloudfour.com-patterns/pull/691#discussion_r421855239)
4. Determine if we still need the Flickr icon

---

- Addresses part of #657
